crates/tests/src/features.rs

```diff
@@ -761,6 +761,53 @@ fn events() {
     })
 }
 
+#[test]
+fn enum_match() {
+    with_executor(&|mut executor| {
+        let harness = deploy_contract(
+            &mut executor,
+            "enum_match.fe",
+            "Foo",
+            &[uint_token(26), uint_token(42)],
+        );
+
+        harness.test_function(
+            &mut executor,
+            "simple_match",
+            &[uint_token(1), uint_token(2)],
+            Some(&uint_token(3)),
+        );
+
+        harness.test_function(
+            &mut executor,
+            "nested_match",
+            &[uint_token(1), uint_token(2)],
+            Some(&uint_token(3)),
+        );
+
+        harness.test_function(&mut executor, "wild_card", &[], Some(&uint_token(0)));
+
+        harness.test_function(&mut executor, "match_in_if", &[], Some(&uint_token(3)));
+
+        harness.test_function(&mut executor, "match_in_loop", &[], Some(&uint_token(15)));
+
+        harness.test_function(
+            &mut executor,
+            "enum_storage",
+            &[uint_token(1), uint_token(2), bool_token(true)],
+            Some(&uint_token(3)),
+        );
+        // harness.test_function(
+        //     &mut executor,
+        //     "enum_storage",
+        //     &[uint_token(1), uint_token(2), bool_token(false)],
+        //     Some(&uint_token(100)),
+        // );
```

Comment:
@g-r-a-n-t 
When these lines are uncommented, the test fails with the message `thread 'features::enum_match' panicked at 'attempt to subtract with overflow', crates/test-utils/src/lib.rs:158:24`.
This seems to be a bug related to `GasReporter` or `evm::StackExecutor`, but I didn't look into it yet.